profiles/bicycle.lua

```diff
@@ -93,8 +93,9 @@ surface_speeds = {
 
 -- these need to be global because they are accesed externaly
 properties.traffic_signal_penalty        = 2
-properties.use_turn_restrictions         = false
 properties.u_turn_penalty                = 20
+properties.max_speed_for_map_matching    = 30/3.6 -- 15.5 km/h is the average speed
```

Comment:
Just to make it clear here: the max speed is like an upper bound over which traces are getting filtered out - you don't want to set an average speed here, otherwise you will throw away half of all traces. Max speed should be an upper bound of what makes sense per profile. (Think of a bike trace with 200 km/h _probably_ makes no sense :P)